src/components/harvestLogs/HarvestLogDrawer.tsx

```diff
@@ -207,15 +207,15 @@ const HarvestLogDrawer = ({
     created: IHarvestLogResponse & { _id: string }
   ) => {
     createHarvestLogCache(created);
-    showAlert(`Harvest Log created successfully`);
+    showAlert(`Harvest Log created successfully`, "error");
```

Comment:
This should be success

Reply:
woops! https://github.com/LangaraVoyagers/project-2/pull/23/commits/a3b62e31197fa092f74f7bdbd2474ede5c01ad86